src/Public/js/nipsweb.librarypane.js

```diff
@@ -10,7 +10,7 @@ var entityMap = {
   "&": "&amp;",
   "<": "&lt;",
   ">": "&gt;",
-  '"': "&quot;",
+  "\"": "&quot;",
```

Comment:
Then blame the syntax highlighter :p
ideally i would use '' for it, but eslint complained, so meh